sdks/python/apache_beam/transforms/ptransform_test.py

```diff
@@ -1146,7 +1146,7 @@ def test_pipeline_checking_pardo_insufficient_type_information(self):
                      e.exception.message)
 
   def test_pipeline_checking_gbk_insufficient_type_information(self):
-    self.p.options.view_as(TypeOptions).type_check_strictness = 'ALL_REQUIRED'
+    self.p._options.view_as(TypeOptions).type_check_strictness = 'ALL_REQUIRED'
```

Comment:
I believe it would be better if `setUp` defined an options like:
```
self.options = PipelineOptions()
self.p = TestPipeline(self.options)
```
then all of this code would use `self.options...` instead of accessing `_options`.